packages/react/src/useSynthql.test.tsx

```diff
@@ -56,6 +56,32 @@ describe('useSynthql tests', () => {
         pagilaServer?.server.close();
     });
 
+    test('Fetching 0 or 1 rows(s) from the Pagila database without `columns()`', async () => {
+        // @@start-example@@ Find a single actor by id without `columns()`
```

Comment:
This is not an easy title to search by. When composing this titles try to think of them as "google queries". 

In this case people would more likely search by something like Select all columns from the actors table

And with regards to the description, the important bit is to mention that if you omit the `select` all selectable fields will be selected by default.